tools/nvqpp/nvq++.in

```diff
@@ -39,10 +39,19 @@ function add_pass_to_pipeline {
 }
 
 function list_targets {
-	ls -I *.cpp -1 ${install_dir}/targets/ | sed -e 's/\.config$//'
+	ls -I *.cpp -1 ${install_dir}/targets/ | grep ".config$" | sed -e 's/\.config$//'
 	exit 0
 }
 
+function list_simulators {
+	for file in $(grep -L "PLATFORM_QPU=" $(ls ${install_dir}/targets/*.config));
+	do
```

Comment:
```suggestion
	for file in $(grep -L "PLATFORM_QPU=" $(ls ${install_dir}/targets/*.config)) ; do
```
since the previous line already had a `;` at the end.